### PR TITLE
qe: Skipping failing tests on WASM

### PR DESCRIPTION
**Pull request by SevInf**
Fix prisma/team-orm#709

### CODE REVIEW

File: query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_12572.rs [hunk 1]
```diff
@@ -26,7 +26,7 @@ mod prisma_12572 {
         .to_owned()
     }
 
-    #[connector_test]
+    #[connector_test(exclude(Postgres("pg.js.wasm", "neon.js.wasm")))]
```
Comment (SevInf) [inline query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_12572.rs:29]: TODO: open an issue for that

Comment (SevInf) [reply query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_12572.rs:29]: https://github.com/prisma/team-orm/issues/645#issuecomment-1845779338

**Comment by janpio** [comment]
One left:
```
failures:

---- new::regressions::prisma_12572::prisma_12572::all_generated_timestamps_are_the_same stdout ----
Used datamodel:
 datasource test {
    provider = "sqlite"
    url = "file:///home/runner/work/prisma-engines/prisma-engines/db/prisma_12572_all_generated_timestamps_are_the_same.db"
    relationMode = "foreignKeys"
    
}

generator client {
    provider = "prisma-client-js"
    previewFeatures = ["deno", "driverAdapters", "fullTextIndex", "fullTextSearch", "metrics", "multiSchema", "nativeDistinct", "postgresqlExtensions", "tracing", "views", "relationJoins"]
}



            model Test1 {
                id String @id
                up1 DateTime @updatedAt
                cr1 DateTime @default(now())
                cr2 DateTime @default(now())
                up2 DateTime @updatedAt
                test2s Test2[]
            }

            model Test2 {
                id String @id
                test1Id String @unique
                test1 Test1 @relation(fields: [test1Id], references: [id])
                cr DateTime @default(now())
                up DateTime @updatedAt
            }
        
thread 'new::regressions::prisma_12572::prisma_12572::all_generated_timestamps_are_the_same' panicked at query-engine/connector-test-kit-rs/query-engine-tests/tests/new/regressions/prisma_12572.rs:56:13:
assertion `left == right` failed
  left: "2023-12-07T18:15:53.785Z"
 right: "2023-12-07T18:15:53.784Z"
```

Clicking "restart" anyway, maybe it goes green.

File: .github/workflows/test-query-engine-driver-adapters.yml [hunk 2]
```diff
@@ -25,24 +25,24 @@ jobs:
...
+          - name: 'planetscale (napi)'
             setup_task: 'dev-planetscale-js'
-          - name: '@prisma/adapter-pg (napi)'
+          - name: 'pg (napi)'
             setup_task: 'dev-pg-js'
-          - name: '@prisma/adapter-neon (ws) (napi)'
+          - name: 'neon (ws) (napi)'
             setup_task: 'dev-neon-js'
-          - name: '@prisma/adapter-libsql (Turso) (napi)'
+          - name: 'libsql (napi)'
```
Comment (Jolg42) [inline .github/workflows/test-query-engine-driver-adapters.yml:34]:
```suggestion
          - name: 'libsql Turso (napi)'
```
to make it similar to the other one

**Pull request merged**